web/gds-user-ui/src/components/Section/__tests__/__snapshots__/VaspVerification.spec.tsx.snap

```diff
@@ -257,7 +257,11 @@ exports[`<Line /> should match snapshot 1`] = `
           </div>
         </div>
         <div
+<<<<<<< HEAD
```

Comment:
@elysee15 I guess you forget to resolve this conflict . would you mind fix that?